automation_tools/satellite6/capsule.py

```diff
@@ -543,8 +545,9 @@ def setup_capsule_content(
     )
     hammer_content_view_publish(content_view_name, organization_id)
     product_id = run(
-        'hammer --csv subscription list --organization-id="{}" '
-        '--search="name=\\"{}\\"" | cut  -d "," -f 8 | grep -vi "ID"',
+        "hammer --csv subscription list --organization-id='{0}' "
+        "--search='name=\"{1}\"' | awk -F, 'NR>1{{print$8}}'"
+        .format(organization_id, product_name),
```

Comment:
switched ```'``` and ```"``` to make it work
plus escaped ```{print$8}``` (to avoid an attempt to parse it by .format())